decentraland/unity-explorer
# Fix: emoji lock input when search

## What does this PR change?
Fix #1572 

- add lock for input  when in search bar

## How to test the changes?

1. Launch the explorer
2. Open emoji search bar
3. Use letters that would trigger shortcuts (U, M, A, S, W, D) -> Player should not move and shortcut should not trigger
4. Close emoji and verify that shortcuts and movement is functional again
5. Try different UX approaches to find possible edge cases when enabling/disabling (opening/closing of emoji and its search bar)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
